<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogLaserControl</class>
 <widget class="QDialog" name="dialogLaserControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Laser Manual Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/main/iconPyMSRun.svg</normaloff>:/main/iconPyMSRun.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QLabel" name="imgLaser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>133</width>
     <height>133</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/main/laser.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Laser Power</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/laser/laserform_close.png</normaloff>:/laser/laserform_close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>:/laser/laserform_on_disabled.png</disabledoff>
     <disabledon>:/laser/laserform_on_disabled.png</disabledon>
     <activeoff>:/laser/laserform_on.png</activeoff>
     <activeon>:/laser/laserform_off.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>31</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sliderEnable">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>53</y>
     <width>22</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sliderLaser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btnClose</zorder>
  <zorder>btnOn</zorder>
  <zorder>sliderEnable</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>imgLaser</zorder>
  <zorder>lblStatus</zorder>
  <zorder>sliderLaser</zorder>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
